ark/utils/segmentation_utils_test.py

```diff
@@ -1,9 +1,42 @@
 import numpy as np
 import xarray as xr
-
+import os.path
+from os import path
 from skimage.measure import regionprops
+import tempfile
+
+from ark.utils import segmentation_utils, data_utils
+
+
+def _generate_channel_xr(fov_num=2, chan_num=5):
+    fovs = ["fov" + str(i) for i in range(fov_num)]
+    channels = ["channel" + str(i) for i in range(chan_num)]
+    output = np.random.randint(0, 20, len(fovs) * 50 * 50 * len(channels))
+    output = output.reshape((len(fovs), 50, 50, len(channels)))
+
+    output_xr = xr.DataArray(output, coords=[fovs, range(50), range(50), channels],
+                             dims=["fovs", "rows", "cols", "channels"])
+
+    return output_xr
 
-from ark.utils import segmentation_utils
+
+def _generate_deepcell_ouput(fov_num=2):
+    fovs = ["fov" + str(i) for i in range(fov_num)]
+    models = ["pixelwise_interior", "watershed_inner", "watershed_outer",
+              "fgbg_foreground", "pixelwise_sum"]
+    output = np.random.rand(len(fovs) * 50 * 50 * len(models))
+    output = output.reshape((len(fovs), 50, 50, len(models)))
+
+    output_xr = xr.DataArray(output, coords=[fovs, range(50), range(50), models],
+                             dims=["fovs", "rows", "cols", "models"])
+
+    label_data = np.zeros((fov_num, 1024, 1024, 1))
+    label_xr = xr.DataArray(
+        label_data,
+        coords=[fovs, range(1024), range(1024), ['whole_cell']],
+        dims=("fovs", "rows", "cols", "compartments")
+    )
```

Comment:
The 1024's can be resized to 50's

Also, I believe `label_data = np.random.randint(0, 100, size=(fov_num, 50, 50, 1))` would also work here, instead of `np.zeros`.  I don't think it matters either way.